### ISSUE TITLE
Make bottom panel tabs more visually distinct

### ISSUE THREAD

**Issue by per1234** [arduino/arduino-ide#609]
When multiple panels are opened in the bottom panel, they are controlled by a tabs UI:

---

There is very little visual contrast between the tabs:

![image](https://user-images.githubusercontent.com/8572152/141963206-a0f84f0d-2e73-4343-bba3-fc97af5419ad.png)

The selected vs unselected tab color is almost the same.

---

The tabs style is not consistent with the sketch tabs style:

![image](https://user-images.githubusercontent.com/8572152/141963323-e9785e8f-4298-4bc6-b6dc-2307958c1c9a.png)

Selected vs unselected sketch tabs have significantly different colors.

**To Reproduce**

1. Compile or upload a sketch to open the "Output" panel
2. Use **Tools > Serial Monitor** to open the "Serial Monitor panel
3. Use the downward pointing triangle menu > New Tab to create a second sketch tab

**Expected behavior**

- Bottom panel tabs are visually distinct from each other
- It is visually obvious which bottom panel tab is selected
- Consistent style of all tab UIs in the IDE

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version: 2.0.0-beta.12-nightly.20211115
Date: 2021-11-15T03:03:14.165Z
CLI Version: 0.19.1 alpha [718bbbf2]

**Comment by PerryBebbington** [comment]
@per1234 
I agree with this but I do not agree that 'Selected vs unselected sketch tabs have significantly different colors'; they are just as bad (well, maybe not quite as bad but there is far too little contrast).
